.github/workflows/cancel.yml

```diff
@@ -0,0 +1,13 @@
+name: Cancel
+on: [push]
+jobs:
+  cancel:
+    name: "Cancel Previous Build"
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          workflow_id: ".github/workflows/build.yml"
+          all_but_latest: true
```

Comment:
Does it cancel other branches builds too ?

Reply:
no